mindmeld/app_manager.py

```diff
@@ -197,7 +199,8 @@ def parse(
 
         allowed_intents, nlp_params, dm_params = self._pre_nlp(params, verbose)
         processed_query = self.nlp.process(
-            query_text=text, allowed_intents=allowed_intents, **nlp_params
+            query_text=text, allowed_intents=allowed_intents, **nlp_params,
+            language=self.language
```

Comment:
`language` is already getting passed in as one of the `**nlp_params`. I think the *fix* that is needed is just to add `params.language` and `params.locale` to the docstring above. If it helpful to make language/locale a permanent property of the app, we should still treat `language` and `locale` in parallel.

Reply:
I think in this case we can have a default language for the app, which can be overridden by the request.